.env.example

```
@@ -1,3 +1,7 @@
-REDDIT_CLIENT_ID=your_client_id_here
-REDDIT_CLIENT_SECRET=your_client_secret_here
-REDDIT_USER_AGENT="script:reddit-mcp-server:v0.1.0 (by u/your_reddit_username)"
+ REDDIT_CLIENT_ID=your_client_id_here
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)